ios/StripeSdk.swift

```diff
@@ -156,6 +156,7 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
                 }
             } else {
                 self.paymentSheet = PaymentSheet(paymentIntentClientSecret: paymentIntentClientSecret, configuration: configuration)
+                self.paymentSheetFlowController = nil
```

Comment:
thanks for this PR! 

I'm thinking it might be cleaner to call `self.paymentSheetFlowController = nil` just once at the top of the `initPaymentSheet` function, what do you think?

Reply:
Yeah, that's is much better. Thanks!